Tests/AsyncHTTPClientTests/TransactionTests.swift

```diff
@@ -567,7 +545,7 @@ actor SharedIterator<Wrapped: AsyncSequence> where Wrapped.Element: Sendable {
 
 /// non fail-able promise that only supports one observer
 @available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
-fileprivate actor Promise<Value> {
+private actor Promise<Value> {
```

Comment:
The change from `fileprivate` to `private` is the result of running `SwiftFormat`